interface/patient_file/front_payment.php

```diff
@@ -482,7 +487,7 @@ function toencounter(enc, datestr, topframe) {
   <td><?php echo text(oeFormatMoney($payrow['amount1'])) ?></td>
  </tr>
  <tr>
-  <td><?php echo xlt('Amount for Past Balance'); ?>:</td>
+  <td><?php if($_REQUEST['radio_type_of_payment']=='pre_payment'){echo xlt('Pre-payment Amount');} else {echo xlt('Amount for Past Balance');} //Added by Sherwin 7/21/2017  ?>:</td>
```

Comment:
PSR2:
``````
  <td>
    <?php
    if ($_REQUEST['radio_type_of_payment']=='pre_payment') {
        echo xlt('Pre-payment Amount');
    } else {
        echo xlt('Amount for Past Balance');
    } //Added by Sherwin 7/21/2017
    ?>
  :</td>